### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by tarjei** [olahol/reactpack#15]
Hi, it would be great if Reactpack could include a story about testing. My suggestion would be the following components:
- Karma
- Mocha
- Phantomjs (optional?)

Then reactpack could get the tests from a directory or a pathspec on the commandline and run them using its standard configuration.

**Comment by olahol** [comment]
Yeah I'm thinking about adding testing functionality this week, thank you @tarjei
